deploy/tests/000024-enable-beta-pools.ts

```diff
@@ -0,0 +1,64 @@
+import { PoolCollection } from '../../components/Contracts';
+import { NetworkSettingsV1 } from '../../components/LegacyContractsV3';
+import { describeDeployment } from '../../test/helpers/Deploy';
+import { DeployedContracts } from '../../utils/Deploy';
+import { NATIVE_TOKEN_ADDRESS } from '../../utils/TokenData';
+import { toCents, toWei } from '../../utils/Types';
+import { expect } from 'chai';
+import { getNamedAccounts } from 'hardhat';
+
+describeDeployment(__filename, () => {
+    let networkSettings: NetworkSettingsV1;
+    let poolCollection: PoolCollection;
+
+    // TODO: make sure to update the limits and the rates before running the script in production
```

Comment:
can probably remove this comment now (and similar comments)

Reply:
👍